src/Symfony/Component/Console/Input/Input.php

```diff
@@ -13,6 +13,7 @@
 
 use Symfony\Component\Console\Exception\InvalidArgumentException;
 use Symfony\Component\Console\Exception\RuntimeException;
+use function array_keys;
```

Comment:
unsure about those imports, but I thought the CS fixer was taking care of those?

Reply:
It's better to prefix your `array_keys` call by `\` to avoid those imports